exchanges/coinbasepro/coinbasepro_wrapper.go

```diff
@@ -2,11 +2,13 @@ package coinbasepro
 
```

Comment:
Just posting this for general PSA: Coinbase is looking to move to a new API system. Currently its docs are lacking, so I'm not recommending we change to it. However, our API implementation is now considered "legacy" in Coinbase's eyes

Reply:
To further this, you can no longer generate new legacy keys 🌞